pkg/edgeview/src/tcp.go

```diff
@@ -283,37 +290,53 @@ func recvServerData(mtype int, message []byte) {
 func setAndStartProxyTCP(opt string) {
 	var ipAddrPort []string
 	var proxySvr *http.Server
+	var kubeport int
 	proxyServerDone := make(chan struct{})
 
 	mappingCnt := 1
 	ipAddrPort = make([]string, mappingCnt)
 
-	var hasProxy bool
+	var hasProxy, hasKube bool
 	var proxyDNSIP string
+	kubenum := edgeviewInstID - 1
 	if strings.Contains(opt, "/") {
-		var gotProxy bool
+		var gotProxy, gotKube bool
 		params := strings.Split(opt, "/")
 		mappingCnt = len(params)
 		ipAddrPort = make([]string, mappingCnt)
 		for i, ipport := range params {
-			gotProxy, proxyDNSIP = getProxyOpt(ipport)
-			if !strings.Contains(opt, ":") && !gotProxy {
+			gotProxy, gotKube, proxyDNSIP = getProxyOpt(ipport)
+			if !strings.Contains(opt, ":") && !gotProxy && !gotKube {
```

Comment:
Is this accurate? The log message said it should be `ip:port` or start with `proxy@`, but it looks like kube is a different format? Where is that format defined and coded?